addons/website_mass_mailing/controllers/website_form.py

```diff
@@ -12,7 +12,10 @@ class WebsiteNewsletterForm(WebsiteForm):
 
     def _handle_website_form(self, model_name, **kwargs):
         if model_name == 'mailing.contact':
-            list_ids = [int(x) for x in kwargs['list_ids'].split(',')]
+            list_ids = kwargs.get('list_ids', None)
+            if not list_ids:
+                return json.dumps({'error': _('Mailing List(s) not found!')})
+            list_ids = [int(x) for x in list_ids.split(',')]
```

Comment:
I see @rdeodoo already commented on this PR, I'll wait for his reaction first :wink: